app/src/main/res/values/strings.xml

```diff
@@ -236,6 +236,7 @@
     <string name="error_reading_qr"> Error while reading QR, make sure you select proper region
     </string>
     <string name="error_fetching_image">Error fetching image </string>
+    <string name="invalid_phn_number">Phone number entered is invalid</string>
```

Comment:
Let's give a concise and short message to the user, update the message to: `Invalid phone number`

Reply:
The string should be "Invalid Phone Number" or "Invalid phone number" (use capital or lowercase p) other changes look good to me :)

Reply:
Changed it to "Invalid Phone Number"